locales/es/json.json

```diff
@@ -976,7 +976,7 @@
     "Update VAT ID": "Actualizar VAT ID",
     "Update your account's profile information and email address.": "Actualice la información de su cuenta y la dirección de correo electrónico.",
     "Updating": "Actualizando",
-    "Uploading files... (:current/:total)": "Uploading files... (:current/:total)",
+    "Uploading files... (:current/:total)": "Subiendo archivos... (:actual/:total)",
```

Comment:
This machine translations modifies the placeholder attribute `:current`, the placeholders must not be translated.